tasmota/language/en_GB.h

```diff
@@ -531,6 +531,10 @@
 #define D_SENSOR_DALI_TX                  "Dali TX"
 #define D_CONFIGURE_DALI                  "DALI"
 
+// xdrv_91_esp32_gnss.ino
```

Comment:
You need to copy this section in all languages files (use search/replace multi-line) to make it at once.